cms/envs/production.py

```diff
@@ -395,32 +395,32 @@ def get_env_setting(setting):
     DATADOG['api_key'] = AUTH_TOKENS['DATADOG_API']
 
 # Celery Broker
-CELERY_ALWAYS_EAGER = ENV_TOKENS.get("CELERY_ALWAYS_EAGER", False)
-CELERY_BROKER_TRANSPORT = ENV_TOKENS.get("CELERY_BROKER_TRANSPORT", "")
-CELERY_BROKER_HOSTNAME = ENV_TOKENS.get("CELERY_BROKER_HOSTNAME", "")
-CELERY_BROKER_VHOST = ENV_TOKENS.get("CELERY_BROKER_VHOST", "")
-CELERY_BROKER_USER = AUTH_TOKENS.get("CELERY_BROKER_USER", "")
-CELERY_BROKER_PASSWORD = AUTH_TOKENS.get("CELERY_BROKER_PASSWORD", "")
-
-BROKER_URL = "{0}://{1}:{2}@{3}/{4}".format(CELERY_BROKER_TRANSPORT,
-                                            CELERY_BROKER_USER,
-                                            CELERY_BROKER_PASSWORD,
-                                            CELERY_BROKER_HOSTNAME,
-                                            CELERY_BROKER_VHOST)
-BROKER_USE_SSL = ENV_TOKENS.get('CELERY_BROKER_USE_SSL', False)
-
-BROKER_TRANSPORT_OPTIONS = {
+CELERY_TASK_ALWAYS_EAGER = ENV_TOKENS.get("CELERY_ALWAYS_EAGER", False)
```

Comment:
Short version: This line is not in lms/envs/production.py because the setting gets automatically pulled from the yaml file.  Update the `lms/envs/bok_choy.yml` file to have the correct variable`CELERY_TASK_ALWAYS_EAGER` set to `True` and that should hopefully fix the issue.

Long version: Having django settings with a different name from the name of the setting in the YAML file is convenient but in the long run confusing and can lead to errors.  Rather than updating this to pull from the old name in config to a new name in settings, you should add the new names and values to the settings files(test,sandbox, stage, prod, edge).  This work could be done as a separate step once everything is live but should be done before we consider this upgrade fully complete.  It will also need a note for the koa release as it will impact open-source operators.